iotedgedev/cli.py

```diff
@@ -1,3 +1,4 @@
+from email.policy import default
```

Comment:
what is this library needed for?

Reply:
sorry I missed that, no idea how that came in. It's not needed

Reply:
I blame GitHub co-pilot 😄 
But, I should have spotted the change before committing it - sorry about that.